python/tk_framework_desktopserver/server.py

```diff
@@ -21,13 +21,15 @@
 
 from .errors import MissingCertificateError, PortBusyError
 
+from .logger import get_logger
+
 
 class Server:
```

Comment:
Now I am curious why we are not deriving from `object`? I think we should always do that and if we don't there should be a *big* comment explaining why.